.github/actions/sign-attestations/src/main.ts

```diff
@@ -50,11 +56,10 @@ async function run(): Promise<void> {
         );
         const bundleStr = JSON.stringify(bundle);
         // We detect path traversal for safeOutputFolder, so this should be safe.
-        const outputPath = path.join(
+        const outputPath = `${path.join(
```

Comment:
I dont think you need this. The following should work, no:
```
const outputPath = path.join(
           safeOutputFolder
           path.basename(file) + ".sigstore",
        );
```

Reply:
npm style doesn't like string concatenation and prefers templating D:

/home/asraa/git/slsa-github-generator/.github/actions/sign-attestations/src/main.ts
  61:11  error  Unexpected string concatenation  prefer-template

But it's cleaner to do similar to what you suggested:
```
        const outputPath = path.join(
          safeOutputFolder,
          `${path.basename(fpath)}.sigstore`);
```

Reply:
Got it, thanks!